content/ch-algorithms/teleportation.ipynb

```diff
@@ -2454,14 +2455,41 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## 3. Simulating the Teleportation Protocol <a id='simulating'></a>"
+    "## 3. Jacklyn sends Vivian Roses: An Analogy <a id='analogy'></a>"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### 3.1 How Will We Test the Protocol on a Quantum Computer? <a id='testing'></a>"
+    "Now that we know the quantum teleportation protocol, I want to take a step back and make the protocol more relatable with an analogy.\n",
+    "\n",
+    "Jacklyn, the first qubit, wants to send roses to Vivian. Those roses are the information stored in the first qubit. In order to send the roses, she gets help from FlowerDelivery, a delivery company, to deliver the roses to Vivian. The company provides the delivery truck and Vivian's information, and together those pieces of information help determine the directions to Vivian's house. The entangled qubit channel is this route, while the entangled qubits are the delivery truck and Vivian.\n",
```

Comment:
I think it's rather confusing that in the original protocol we have Alice _send qubits_ to Bob, while here _Jacklyn and Vivian are the qubits_ trying to send each other information.

Reply:
I agree and am willing to make the analogy more relatable to the Quantum Teleportation Protocol. Thank you for the comment.